src/components/Disputes/DisputeEvidences.js

```diff
@@ -153,10 +153,10 @@ const EvidenceContent = React.memo(function EvidenceContent({
           >
             {(() => {
               if (addressesEqual(plaintiff, submitter)) {
-                return 'In favor'
+                return 'Against'
               }
               if (addressesEqual(defendant, submitter)) {
-                return 'Against'
+                return 'In favor'
```

Comment:
Let's check with @dizzypaty on this text. We should use the same text as we do on the voting buttons.

Reply:
@rperez89 , @sohkai  – I'd say let's remove this field `Argument` from the evidence altogether. At least for now until we have a more clear solution. It's creating more confusion than helping actually.